web/public/js/main.js

```diff
@@ -0,0 +1,73 @@
+(function() {
+    'use strict';
+    var socket = io();
+
+    // socket.emit('connection', socket.id);
+
+    socket.on('updated data', function(data) {
+        updateChart(data)
```

Comment:
The same goes for ES6. Either use it across the entire project or don't. I can set up browserify with babel if you would like.